psychopy_pixx/routines/viewpixx/__init__.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+
+
+
+from pathlib import Path
+from psychopy.localization import _translate
+from psychopy.experiment import Param
+from psychopy.experiment.routines import BaseStandaloneRoutine
+
+
+
+
+
+
+class ViewpixxSetupRoutine(BaseStandaloneRoutine):  
+    """A class for the viewpixx device
+    """
+    categories = ['Responses']
+    targets = ['PsychoPy']
+    iconFile = Path(__file__).parent / 'viewpixx.png'
+    tooltip = _translate('Viewpixx: Setup a Viewpixx (Should be the first routine of the experiment)')
+    plugin = "psychopy-pixx"
+    
+    def __init__(self, exp, name='viewpixx', vmode='M16', scanBackLight=False, disabled=False):
+        BaseStandaloneRoutine.__init__(self, exp, name = name, disabled=disabled)
+        self.url = ""
```

Comment:
What is this url attribute for?

Reply:
Cool! Could you point it to the README?

https://github.com/wichmann-lab/psychopy-pixx#viewpixx-monitor
https://github.com/wichmann-lab/psychopy-pixx#responsepixx-button-box